config.dev.toml

```diff
@@ -75,11 +75,6 @@ Timeout = 300
     # Env override: ATHENS_REDIS_QUEUE_ADDRESS
     RedisQueueAddress = ":6379"
 
-    # Flag to turn off Proxy Filter middleware
-    # Defaults to true
-    # Env override: PROXY_FILTER_OFF
-    FilterOff = true
```

Comment:
This means the default behavior is that it's always ON since this config file's FileFile property is not empty. Can you ensure that property is an empty string by default. Thanks!

Reply:
Just removed those changes.

Reply:
Fixed!

Reply:
@manugupt1 not the FilterOff property, but the FilterFile in `config.dev.toml` must be an empty string now that we don't have an off bool, which is a good thing :)